vtex-apps/store-header
# Make store-header great again

#### What is the purpose of this pull request? What problem is this solving?
🎉 No more classes: entirely using [hooks](https://reactjs.org/docs/hooks-intro.html)
🔥 Faster [spring-based](https://www.react-spring.io/) animations that do not trigger re-rendering
🏗 Updated project structure, to avoid a mess
📱 Fix Resize problems
🌈 Make icons, labels, logo size, and animation of collapsible content configurable through Header
👨🏾‍💻 Updated docs

#### How should this be manually tested?
Use [this workspace](https://makeheadergreatagain--storecomponents.myvtex.com/).
Compare it to [master](https://storecomponents.myvtex.com/) and see the difference.

#### Types of changes
